pangeo_forge_recipes/writers.py

```diff
@@ -99,19 +99,20 @@ def write_combined_reference(
     full_target: FSSpecTarget,
     concat_dims: List[str],
     output_file_name: str,
-    target_options: Optional[Dict] = {"anon": True},
-    remote_options: Optional[Dict] = {"anon": True},
-    remote_protocol: Optional[str] = None,
     refs_per_component: int = 1000,
     mzz_kwargs: Optional[Dict] = None,
 ) -> zarr.storage.FSStore:
     """Write a kerchunk combined references object to file."""
     file_ext = os.path.splitext(output_file_name)[-1]
     outpath = full_target._full_path(output_file_name)
 
+    import ujson  # type: ignore
+
     # If reference is a ReferenceFileSystem, write to json
     if isinstance(reference, fsspec.FSMap) and isinstance(reference.fs, ReferenceFileSystem):
-        reference.fs.save_json(outpath, **remote_options)
+        reference_meta = reference.translate()
```

Comment:
this is wrong but since this PR is gonna be changed I'm gonna leave it as is for now